### PR TITLE
Add message editing functionality

### PR DESCRIPTION
**Pull request by samuelyallop-pusher**
Adds functions for (upcoming) message editing endpoints.

----

- [x] CHANGELOG updated if relevant?

### CODE REVIEW

File: src/Chatkit.php [hunk 1]
```diff
@@ -1382,10 +1488,10 @@ function($curl, $header) use (&$headers)
 ];
 
 const MESSAGE_ID = [ 'message_id' =>
-                        [ 'type' => 'integer',
-                          'missing_message' =>
-                          'You must provide the message ID'
-                        ]
+                     [ 'type' => 'int',
+                       'missing_message' =>
+                       'You must provide the ID of the message to edit'
```
Comment (minaorangina) [inline src/Chatkit.php]: This is being used by `fetchMultipartMessage` as well. Either the error message should remain generic, or we have two separate ones.

**Review by minaorangina — CHANGES_REQUESTED** [review]
Would be good to include tests for an unknown message id/room id

File: src/Chatkit.php (hunk 1 above)
Comment (samuelyallop-pusher) [reply src/Chatkit.php]: Yep. Bad merge.

Comment (samuelyallop-pusher) [reply src/Chatkit.php]: 9c3962c

**Comment by samuelyallop-pusher** [comment]
> Would be good to include tests for an unknown message id/room id

See comment in chatkit-server-ruby. I don't think those tests belong in the SDKs.

**Comment by minaorangina** [comment]
It's good to test that SDKs handle these scenarios gracefully and don't just explode. Having said that, many of the SDK tests just end up checking if there's a 404 in an error object, which I agree is redundant. I won't die on this hill, I'll let you make the call

**Review by minaorangina — APPROVED** [review]

**Pull request merged**